frontend/src/types.ts

```diff
@@ -13,37 +13,28 @@ import { PluginConfigSchema } from '@posthog/plugin-scaffold'
 import { PluginInstallationType } from 'scenes/plugins/types'
 import { ViewType } from 'scenes/insights/insightLogic'
 
+export type AvailableFeatures = 'zapier' | 'organizations_projects' | 'google_login' | 'dashboard_collaboration'
+
 export interface UserType {
-    anonymize_data: boolean
-    distinct_id: string
+    id: string
```

Comment:
This needs `uuid` too I think?

Reply:
Same as above, we actually send the `uuid` as `id` and no longer send the internal PK in the payload.

Reply:
`UserBasicType` has both `id` and `uuid` though? I think if the DB fields are not renamed but in API they are, it's kind of confusing, though I agree with the sentiment of sending only `uuid`

Reply:
Makes sense, this could lead to confusion. As this requires a backend change too, will merge this and then update to have the attribute named uuid and avoid confusion.